homeassistant/components/emulated_hue/config.py

```diff
@@ -166,6 +205,27 @@ def get_entity_name(self, state: State) -> str:
 
         return state.attributes.get(ATTR_EMULATED_HUE_NAME, state.name)
 
+    @cache  # pylint: disable=method-cache-max-size-none
+    def get_exposed_states(self) -> list[State]:
+        """Return a list of exposed states."""
+        state_machine = self.hass.states
+        states: list[State] = []
```

Comment:
Maybe move this line after the next if block, before the entities iteration?

Reply:
Yeah that makes more sense.  I originally hard it the other way and rearranged it to reduce indent

Reply:
2166a895e5ddb1ef11799aedb9eadfd37c5dbb8c